starter-code/functions-and-arrays.js

```diff
@@ -34,6 +54,12 @@ console.log(total);
 
 // Calculate the Average
 function averageNumbers (array) {
+  
+
+  var sum = sumArray(array);
+  
+  return sum / array.length;
```

Comment:
You can skip the var declaration and do something like:

return sumArray(array) / array.length;